packages/manager/modules/manager-pci-common/vite.config.mjs

```diff
@@ -0,0 +1,76 @@
+import react from '@vitejs/plugin-react';
+import { defineConfig } from 'vitest/config';
+import dts from 'vite-plugin-dts';
+import path from 'path';
+import tailwindcss from 'tailwindcss';
+import { getBaseConfig } from '@ovh-ux/manager-vite-config';
+import { visualizer } from 'rollup-plugin-visualizer';
+import * as packageJson from './package.json';
+
+const baseConfig = getBaseConfig({});
+const pathSrc = path.resolve(__dirname, 'src');
+
+export default defineConfig({
+  ...baseConfig,
+  resolve: {
+    alias: {
+      '@/': pathSrc,
+    },
+  },
+  plugins: [
+    react(),
+    dts({
+      root: __dirname,
+      insertTypesEntry: true,
+      outDir: 'dist/types',
+    }),
+    visualizer({
+      filename: 'stats/bundle-analysis.html',
+    }),
```

Comment:
to remove ?

Reply:
IMO, we should keep it to have some stats each time the bundle is built.  
If somebody adds or remove dependencies, we should keep an eye on the produces stats to see if the package has to be external or not.

It only generates a `stats` folder that is not embedded in the `dist`,  so there is no impact on the performance and the bundle size.

Reply:
I think we dont use it on a day-to-day basis. So, to make the global node_modules light, i think we can install it in local when it's required and do the analysis.

Reply:
Ok, I comment the plugin usage and remove the devDependency, so it could be easily used if needed